### PR TITLE
Add ChillCenter

### PR DESCRIPTION
**Pull request by poloputoamo**

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1058,5 +1058,12 @@
         "addresses": [
             "alpha.rip"
         ]
+    },
+           {
```
Comment (GrowlyX) [inline servers.json:1062]: Please fix this bracket, you also do not need to add the `play.` subdomain.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Comment by polo-1245-oficial** [comment]
?

File: servers.json [hunk 2]
```diff
@@ -1059,11 +1059,18 @@
             "alpha.rip"
         ]
     },
-           {
+       {
```
Comment (GrowlyX) [inline servers.json:1062]: This is still the incorrect bracket formatting

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 2 above)
Comment (polo-1245-oficial) [reply servers.json]: Ready

**Review by GrowlyX — APPROVED** [review]

**Pull request merged**